Templates & How-Tos for Tutorials & Dev Tools Pages (#461)
protocol/launchpad: templates/README.md

## How to Create a Tutorial

Definitions
* Module: A set of several step-by-step modules
* Tutorial: An individual page in a set of tutorials

  ![Tutorials & Module](tutorials-modules.png)

### Create a new Tutorial
* First, you will need to create a markdown file (or a directory containing a markdown file, if the tutorial has images) in the `/content/en/tutorials` [directory](https://github.com/protocol/launchpad/tree/main/content/en/tutorials) in the root of the repo.
  * Add a tutorial to an existing tutorial module, by simply copy-pasting the tutorials template into the module folder where you will place the file.
       `cp templates/tutorial-temp.md /content/en/tutorials/<module-name>/<tutorial-name.md>`

   * Edit the header/ metadata in the new tutorial:
     Update the `parent:` field in the header, with the identifier from the `config/_default/menus/menus.en.toml` [file](https://github.com/protocol/launchpad/blob/main/config/_default/menus/menus.en.toml):
      ```
     title: "Title (Tutorial)"
     description: "Deep/Shallow Dive Tutorial – Subtitle"
     draft: false
     menu:
       tutorials:
         parent: "<identifier of tutorial module>"
     ```

>      * For example, to add to the **Introduction to IPFS** module, you would replace`"<identifier of tutorial module>"`that with `"tutorials-ipfs"` found on here in the [menus.en.toml file](https://github.com/protocol/launchpad/blob/main/config/_default/menus/menus.en.toml#L159)

Reviewer: ```suggestion
     * For example, to add to the **Introduction to IPFS** module, you would replace `"<identifier of tutorial module>"` with `"tutorials-ipfs"` found on here in the [menus.en.toml file](https://github.com/protocol/launchpad/blob/main/config/_default/menus/menus.en.toml#L159)
```

<br>
  ![Tutorial Identifier](tutorial-identifier.png)

>      * To add the tutorial within a curriculum module as well, under the `menu` field, add an additional two lines with a `curriculm` field witht he correct file from [menus.en.toml](https://github.com/protocol/launchpad/blob/main/config/_default/menus/menus.en.toml#L91):

Reviewer: ```suggestion
     * To add the tutorial within a curriculum module as well, under the `menu` field, add an additional two lines with a `curriculm` field with the correct identifier from [menus.en.toml](https://github.com/protocol/launchpad/blob/main/config/_default/menus/menus.en.toml#L91):
```

       ```
       menu:
         tutorials:
           parent: "tutorials-ipfs"
         curriculum:
           parent: "curriculum-ipfs"
       ```

> 

Reviewer: ```suggestion
If your tutorial does not fit in any of the current categories, then you can create a new category module:
```
